styles/styles.css

```diff
@@ -1072,6 +1257,90 @@ body {
     font-size: 14px;
   }
 
+  .reviews {
+    order: 2;
+    background-color: var(--background-main);
+  }
+
+  .reviews__images {
+    margin-bottom: 12px;
+  }
+
+  .slider-reviews {
+    display: none;
+  }
+
+  .reviews__kingkong {
```

Comment:
Don't use names of images in names of classes, you can use abstract name and pseudoclasses (:last-child, nth-child, etc)